client/components/token-field/index.jsx

```diff
@@ -476,6 +508,18 @@ var TokenField = React.createClass( {
 		}
 	},
 
+	_valueContainsToken( token ) {
+		let search = find( this.props.value, ( item ) => {
```

Comment:
My understanding is that `token` can not be an object here because we are adding a new token from within `TokenField`. Transforming a token to be an object occurs outside of `TokenField`.

Is it confusing? Perhaps a bit. Off of the top of my head, I'm not sure what will make it easier to understand.